### PR TITLE
integration: Use correct type for netns.

### PR DESCRIPTION
**Pull request by eiffel-fl**
Fixes: 0f81d1d42fe0 ("integration: Add test for run trace sni")

### CODE REVIEW

**Review by mqasimsarfraz — APPROVED** [review]
LGTM! Since I made some adjustments, I will wait for another approval before merging it.

**Review by mauriciovasquezbernal — APPROVED** [review]
I tested it locally, before this PR the tests failed, now it works fine. Thanks you both for this fix!

**Review by burak-ok — APPROVED** [review]
LGTM, too.

**Pull request merged**
